oc-chef-pedant/lib/pedant/platform.rb

```diff
@@ -195,18 +195,18 @@ def create_client(name, org = self.test_org)
       payload = { "name" => clientname }
 
       if server_api_version == 0
-        r = post(api_url('/clients'), org.validator, :payload => payload)
+        r = post(api_url('/clients', org), org.validator, :payload => payload)
```

Comment:
If you wanna do a test with multiple orgs, you have to make one :) This code tried to supported that, but was wrong.